src/lib/pages/backoffice/Actions/CheckIn/ItemsSearch/state/actions.js

```diff
@@ -0,0 +1,174 @@
+import { itemApi } from '@api/items';
+import { loanApi } from '@api/loans';
+import { documentApi } from '@api/documents';
+import {
+  sendErrorNotification,
+  sendSuccessNotification,
+  addNotification,
+} from '@components/Notifications';
+import { toShortDate } from '@api/date';
+
+export const IS_LOADING = 'itemsSearch/IS_LOADING';
+export const SUCCESS = 'itemsSearch/SUCCESS';
+export const HAS_ERROR = 'itemsSearch/HAS_ERROR';
+export const QUERY_STRING_UPDATE = 'itemsSearch/QUERY_STRING_UPDATE';
+export const LOAN_IS_LOADING = 'itemLoansSearch/IS_LOADING';
+export const LOAN_SUCCESS = 'itemLoansSearch/SUCCESS';
+export const LOAN_HAS_ERROR = 'itemLoansSearch/HAS_ERROR';
+export const LOAN_RETURN = 'itemLoansSearch/LOAN_RETURN';
```

Comment:
I would name this action a bit differently, when I was reading the code my immediate thought was that it means you are returning the loan which might be misunderstood for the checkin itself. `MULTIPLE_RESULTS` / `MULTIPLE_LOAN_RESULTS`? It would show that it is an exceptional case...